src/Inciter/FaceData.C

```diff
@@ -0,0 +1,67 @@
+// *****************************************************************************
+/*!
+  \file      src/Inciter/FaceData.C
+  \copyright 2012-2015, J. Bakosi, 2016-2018, Los Alamos National Security, LLC.
+  \details   Face-data used only in discontinuous Galerkin discretization scheme
+  \see       FaceData.h for more info.
+*/
+// *****************************************************************************
+
+#include "Reorder.h"
+#include "Vector.h"
+#include "DerivedData.h"
+#include "Inciter/InputDeck/InputDeck.h"
+#include "FaceData.h"
+
+namespace inciter {
+
+extern ctr::InputDeck g_inputdeck;
+
+} // inciter::
+
+using inciter::FaceData;
+
+FaceData::FaceData(
+  const std::vector< std::size_t >& conn,
+  std::size_t nbfac_complete,
+  const std::map< int, std::vector< std::size_t > >& bface,
+  const std::vector< std::size_t >& triinpoel_complete ) : m_bface( bface )
+// *****************************************************************************
+//  Constructor
+//! \param[in] conn Vector of mesh element connectivity owned (global IDs)
+//!   mesh chunk we operate on
```

Comment:
Please add `\param` doxygen comments for the rest of the constructor arguments as well.